snow/validators/test_retriever.go

```diff
@@ -0,0 +1,14 @@
+package validators
```

Comment:
Also I just realized that those mocks are exposed in the package. They should not. This means that anyone using the `validators` package will see the mocks in auto-complete, and be able to use them in normal code.

They should either be defined in a `mocks_test.go` file, or even better would be in a package called `validators_test` within the same directory.

Reply:
This is based on how it was done by Avalanche before, so I agree with his choice to keep it like this.